packages/web-react/src/components/TextField/__tests__/TextField.test.tsx

```diff
@@ -106,7 +106,8 @@ describe('TextField', () => {
       expect(element).toHaveClass('TextField--fluid');
     });
 
-    it.each([['success'], ['warning'], ['error']])('should have %s classname', (validationState) => {
+    // @deprecated Will be removed in next major version.
+    it.each([['success'], ['warning'], ['danger'], ['error']])('should have %s classname', (validationState) => {
```

Comment:
there should be a validation state dictionary test. As there is for colors and sizes.

Reply:
fixed